### DISCUSSION TITLE
TV support

### DISCUSSION THREAD

**Discussion by EthanMendel** [nVuln/homebridge-lg-thinq#217] [Q&A] [answered]
I connected my ThinQ account and have a TV within my ThinQ app, but the console says

`[LGThinQ] No ThinQ devices in your account.`

Are TVs not supported in this plugin?

**Comment by nVuln** [comment, marked as answer]
nop, TV is not covered by this plugin because it don't use thinq api, it's webos, please try using `homebridge-webos-tv`

**Comment by town3r** [comment]
How old is your TV?
Does it not have native HomeKit Support?
Is that the message you see if you have debug enabled?

Also it seems this plugin is really only for the following devices atm.

- Refrigerator	
- Air Purifier	
- Washer & Dryer
- Dishwasher
- Dehumidifier	
- AC

**Comment by EthanMendel** [comment]
🤦🏻‍♂️it has native support tucked away somewhere

**Reply by town3r** [reply]
https://youtu.be/aNpCVuuuVeQ
